### PR TITLE
feat: add  RestClient

### PR DESCRIPTION
**Pull request by dingbo8128**

### CODE REVIEW

File: taosrest/tdengineclient/restclient.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+        Parameters
+        -----------
+        q : SQL statement to execute. Can't be USE statement since RESTful api is stateless.
+
+        Example of Returns
+        -------
+        ```json
+        {
+        "status": "succ",
```
Comment (zitsen) [inline taosrest/tdengineclient/restclient.py:54]: Json indent seems error.

File: tests/test_restful_api.py [hunk 2]
```diff
@@ -40,3 +42,21 @@ def test_server_version():
     assert resp["rows"] == 1
     assert len(resp["column_meta"]) == 1
     assert len(resp["data"]) == 1
+
+
+def test_wrong_sql():
+    """
```
Comment (zitsen) [inline tests/test_restful_api.py:48]: pytest cause error with builtin httpd (not taosadapter)

**Comment by dingbo8128** [comment]
I'll merge it firstly, so you can see all changes as a whole.

File: tests/test_restful_api.py (hunk 2 above)
Comment (dingbo8128) [reply tests/test_restful_api.py:48]: > pytest cause error with builtin httpd (not taosadapter)

I'll add this limitation to document.

**Pull request merged**